bundles/org.openhab.ui/web/src/components/config/editor/hint-rules.js

```diff
@@ -144,9 +149,18 @@ function buildModuleStructure (cm, moduleType) {
   return ret
 }
 
+function hintSceneItems (cm, line, parentNr) {
```

Comment:
> We have to add Enter to the extraKeys of CodeMirror in script-editor.vue

Yes, agreed. That could be valuable, if there are no immediate drawbacks. This can be handled in another PR.

Reply:
Great. I think I can take care of that and try to fix the problem that page/widget editor is reaaally slow in iPadOS in this PR as well.